docs/introduction.mdx

```diff
@@ -121,8 +121,7 @@ They can be used to:
 We currently have adapters for the most popular JavaScript frameworks:
```

Comment:
If you allow me a small nitpick: if we add Laravel now, we should remove "JavaScript" here 🙂